packages/core/legos/package.json

```diff
@@ -137,6 +137,7 @@
     "@chainlink/uniswap-v2-adapter": "1.0.0",
     "@chainlink/uniswap-v3-adapter": "1.0.0",
     "@chainlink/upvest-adapter": "1.0.0",
+    "@chainlink/us-census-adapter": "*",
```

Comment:
Thanks for making this PR! We recently made some changes here where we pin the version, would you mind doing this here as well?

Reply:
Just updated this! Thanks